src/BeneficiaryBundle/Controller/HouseholdController.php

```diff
@@ -446,6 +446,33 @@ public function removeAction(Household $household)
         return new Response($json);
     }
 
+    /**
+     * @Rest\Post("/delete-households")
```

Comment:
Non, c'est pas le format qu'on a
/households/delete ?

Reply:
Non parce que si on fait ça il croit que c'est households/{id} et il dit not found

Reply:
Si on précise dans households/{id} que id c'est un nombre ça devrait marcher non?

Reply:
Non j'ai eu le problème plein de fois par le passé je crois pas que ça marche